lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -88,6 +88,22 @@ public static function find(array $filters = []) {
   private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
+
+    $northstar_response = dosomething_northstar_get_northstar_user($data->uid);
```

Comment:
yeah, if all we need is the NS id and that's available in Drupal, lets not go to NS for that each time @_@ Add it to the query in the signups as something to grab along w/ the singup info so you have it in the results as you're parsing through them in the `build()`